src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -782,4 +794,197 @@ def __set_errors_json(self, error_count_by_operation, errors_by_operation):
             "details": errors_by_operation,
             "message": message
         }
+
+    def __set_error_detail(self, error_code, message):
+        formatted_message = self.__ensure_error_message_restriction_compliance(message)
+        return {
+            "code": str(error_code),
+            "message": str(formatted_message)
+        }
     # endregion
+
+    # region - Patch Truncation
+    def log_truncated_removed_packages_if_any(self):
+        if not len(self.__assessment_truncated_removed) == 0:
+            self.composite_logger.log_debug("Assessment truncated removed packages : {0}".format(self.__assessment_truncated_removed))
```

Comment:
nit: change logs statement to: "Packages under PatchAsssessmentSummary were truncated due to size restrictions. Packages removed from the list are: {0}"